libs/paywall/website/src/lib/use-paywall-bypass-token.ts

```diff
@@ -0,0 +1,25 @@
+import {useEffect} from 'react'
+import {useRouter} from 'next/router'
```

Comment:
Unfortunately, the website builder shouldn't rely on the framework implementation directly (removes vendor lock in of Next).

Next actually allows and suggests to use the history API directly: https://nextjs.org/docs/app/getting-started/linking-and-navigating#windowhistoryreplacestate